liblib/CMakeLists.txt

```
@@ -91,8 +91,8 @@ ENDIF()
 IF(Zstd_FOUND)
 SET(TLL_LIBS ${TLL_LIBS} ${Zstd_LIBRARIES})
 ENDIF()
-IF(Bz2_FOUND)
-SET(TLL_LIBS ${TLL_LIBS} ${Bz2_LIBRARIES})
+IF(BZIP2_FOUND)
+SET(TLL_LIBS ${TLL_LIBS} ${BZIP2_LIBRARIES})
```

The `FindBzip2` module defines an imported library, so might be better to use that, as it will take care of the header locations too:

```suggestion
SET(TLL_LIBS ${TLL_LIBS} BZip2::BZip2)
```

(and below on the `buildplugin` line)

---

IMO it doesn't belong into TLL_LIBS at all: It is only used in the bzip2 "MODULE".